README.md

```diff
@@ -34,6 +34,7 @@ View [the style guide](https://styleguide.egghead.io) for visual examples and co
 - `yarn` to install latest packages
 - `yarn dev:styleguide` to develop using the style guide
 - `yarn dev:library` to develop the library
+- `yarn verify` check linting and builds work
```

Comment:
This runs in CI; you can also run it locally if you want. I would recommend installing an ESLint plugin for  your editor as well to see lint errors inline. Note that linting has no local config - it is using CRA config which only fails on errors, no stylistic lint rules. So in the future this will prevent a lot of the errors I fixed in this PR from getting published (broken imports, unused variables etc.).